app/src/editor/canvas/components/ModuleHelp.jsx

```diff
@@ -22,23 +26,53 @@ export default class ModuleHelp extends React.Component {
     }
 
     async load() {
-        const { moduleId } = this.props
-        const help = await services.moduleHelp({
-            id: moduleId,
-        })
+        const { moduleId, moduleName } = this.props
```

Comment:
Should probably dump existing state immediately on starting to load, otherwise it can show the wrong help while loading.